packages/studio-plugin/tests/__fixtures__/LayoutFile/BasicLayout.tsx

```diff
@@ -1,6 +1,5 @@
 import "./index.css";
 import ComplexBanner from "../ComponentFile/ComplexBanner";
-import "@yext/search-ui-react/index.css";
```

Comment:
if we left this import in, would the tests break?

Reply:
I just made it so that node module imports will work, so added them back!